test/Microsoft.Azure.ServiceBus.UnitTests/RetryTests.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.UnitTests
+{
+    using System;
+    using System.Diagnostics;
+    using System.Security.Authentication;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class RetryTests
+    {
+        [Fact]
+        void RetryPolicyDefaultShouldBeRetryExponential()
+        {
+            var retry = RetryPolicy.Default;
+            Assert.True(retry is RetryExponential);
+        }
+
+        [Fact]
+        void RetryExponentialShouldRetryTest()
+        {
+            // Tuple<ExceptionType, CurrentRetryCount, ShouldRetry>
+            Tuple<Exception, int, bool>[] listOfExceptions =
```

Comment:
Could you please switch to `[Theory][MemberData]` approach similar to what is already done [here](https://github.com/Azure/azure-service-bus-dotnet/blob/f2a4b1bfa7f4073f83210ad2e66b06366e147981/test/Microsoft.Azure.ServiceBus.UnitTests/QueueClientTests.cs#L19)? It will make the test easier to follow and see what fails individually, rather than the whole bunch.